From 18259bb5b09470824a5b5dfb3e83494ffecc27d5 Mon Sep 17 00:00:00 2001
From: Lucas Alvares Gomes <lucasagomes@gmail.com>
Date: Wed, 3 Jul 2013 18:27:34 +0100
Subject: [PATCH 1/1] Open heat-api service port on Fedora

Allow heat-api to be accessed without logging onto bootstrap node

Change-Id: Id53b5e6985f066f4ed810bb2ee1f73fe6b3a4e1b
---
 elements/boot-stack/first-boot.d/97-fedora-iptables | 1 +
 1 file changed, 1 insertion(+)

diff --git a/elements/boot-stack/first-boot.d/97-fedora-iptables b/elements/boot-stack/first-boot.d/97-fedora-iptables
index c99ccab..c85a3bf 100755
--- a/elements/boot-stack/first-boot.d/97-fedora-iptables
+++ b/elements/boot-stack/first-boot.d/97-fedora-iptables
@@ -18,6 +18,7 @@ if [[ "Fedora" = $DISTRO ]]; then
         iptables -I INPUT -p tcp --dport 9292 -j ACCEPT
         iptables -I INPUT -p tcp --dport 5000 -j ACCEPT
         iptables -I INPUT -p tcp --dport 35357 -j ACCEPT
+        iptables -I INPUT -p tcp --dport 8004 -j ACCEPT
 
         # DHCP, TFTP
         iptables -I INPUT -m udp -p udp --dport 67 -j ACCEPT
-- 
1.8.1.4

